### PR TITLE
feat: add topics caching, resolve did keys

### PR DESCRIPTION
**Pull request by hejkerooo**

### CODE REVIEW

File: apps/dsb-client-gateway-scheduler/src/app/modules/topic/topic.module.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { Module } from '@nestjs/common';
+import { TopicRefreshService } from './service/topic-refresh.service';
+import {
+  CronRepositoryModule,
+  TopicRepositoryModule,
+} from '@dsb-client-gateway/dsb-client-gateway-storage';
```
Comment (vikaskumr) [inline apps/dsb-client-gateway-scheduler/src/app/modules/topic/topic.module.ts:6]: as you are renaming everything with ddhub then I will say we can rename `@dsb-client-gateway` to `@ddhub-client-gateway`

Comment (vikaskumr) [inline apps/dsb-client-gateway-scheduler/src/app/modules/topic/topic.module.ts:6]: as you are renaming everything with ddhub then I will say we can rename `@dsb-client-gateway` to `@ddhub-client-gateway`

File: libs/ddhub-client-gateway-enrolment/src/lib/ddhub-client-gateway-enrolment.module.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { forwardRef, Module } from '@nestjs/common';
+import { EnrolmentService } from './service/enrolment.service';
+
+import { ClaimListenerService } from './service/claim-listener.service';
+import { RoleListenerService } from './service/role-listener.service';
+import { EnrolmentRepositoryModule } from '@dsb-client-gateway/dsb-client-gateway-storage';
```
Comment (vikaskumr) [inline libs/ddhub-client-gateway-enrolment/src/lib/ddhub-client-gateway-enrolment.module.ts:6]: same everywhere if we can as we are trying to remove dsb to ddhub

Comment (vikaskumr) [inline libs/ddhub-client-gateway-enrolment/src/lib/ddhub-client-gateway-enrolment.module.ts:6]: same everywhere if we can as we are trying to remove dsb to ddhub

File: apps/dsb-client-gateway-scheduler/src/app/modules/topic/topic.module.ts (hunk 1 above)
Comment (hejkerooo) [reply apps/dsb-client-gateway-scheduler/src/app/modules/topic/topic.module.ts:6]: sure I can do it but in the next PR - don't want to add additional mess

**Review by vikaskumr — APPROVED** [review]

**Review by driamend — APPROVED** [review]

**Pull request merged**